package.json

```
@@ -30,16 +30,17 @@
   },
   "scripts": {
     "clean": "lerna clean",
-    "cover": "lerna run cover",
+    "cover": "yarn workspaces foreach -A run cover",
     "docs": "typedoc",
     "docs:markdown-for-agoric-documentation-repo": "typedoc --plugin typedoc-plugin-markdown --tsconfig tsconfig.build.json",
-    "update": "lernaupdate --dedupe",
+    "update": "yarn upgrade-interactive",
```

I don't think anyone uses this, and if they do they can cope with it being removed. It's not run programmatically.

To lower the risk this could be,
```suggestion
    "update": "echo DEPRECATED use yarn's upgrade-interactive or dedupe",
```